### ISSUE TITLE
Do you intend to export this Role?

### ISSUE THREAD

**Issue by JJ** [jonathanstowe/Staticish#1]
In that case, maybe it should be documented alongside? Even minimally?

https://github.com/jonathanstowe/Staticish/blob/05bb64be61ff4e758fade3b1355178a7963090dd/lib/Staticish.pm#L50

**Comment by jonathanstowe** [comment]
No. it is an _"implementation detail"_, and doesn't have anything that could be useful on its own - it could be a `my` role but I got it in my head that would be a problem when I first made it , (though it doesn't seem to be so may do that for a subsequent release.)
